packages/yoroi-extension/app/components/wallet/staking/dashboard/StakingDashboard.js

```diff
@@ -64,6 +68,8 @@ export type GraphData = {|
 
 type Props = {|
   +graphData: GraphData,
+  +currentTheme: Theme,
```

Comment:
Since you are only using the `currentTheme` to decide whether to show the revamp announcement, it is not necessary to pass in the theme but only a flag to indicate whether the current theme is revamp. This is the "Law of Demeter" of software design.